pom.xml

```diff
@@ -241,6 +241,8 @@
                     <artifactSet>
                         <includes>
                             <include>org.yaml:snakeyaml:*:*</include>
+                            <include>com.github.mifmif:generex:*:*</include>
+                            <include>dk.brics.automaton:automaton:*:*</include>
```

Comment:
Why not just include all dependencies? It's unlikely we'll add more perhaps, but if we would, we'd need to remember to update this too.

Reply:
anyway you need to update relocation part individually , so it will not solve the issue